### DISCUSSION TITLE
A small appearance issue

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#27996] [Other Feature Feedback, Questions, & Ideas]
A gap is missing on icon and text. That is gist by the way.

![IMG_20220802_083725](https://user-images.githubusercontent.com/71941576/182300600-785a0d63-9af7-428b-aa76-6488d2064af6.jpg)

**Comment by jfullstackdev** [comment]
what is that gap ? it's the icon for revisions, I have the same view

![image](https://user-images.githubusercontent.com/47092464/182303666-5063bc21-75a2-4223-acb2-d5c22d155a18.png)

**Reply by Neodevils** [reply]
What I mean by "gap" is, "white space" between icon and text on revisions. Check all other sections on the whole GitHub. You will see there's a white space between icon and section but on revisions there's not.

**Reply by jfullstackdev** [reply]
hopefully let's wait for the Staff members to see this

**Reply by Neodevils** [reply]
Let's wait 🙌🏻

**Reply by queenofcorgis** [reply]
Added a label so this gets to the appropriate team, thanks for flagging!

**Reply by Neodevils** [reply]
> Added a label so this gets to the appropriate team, thanks for flagging!

@queenofcorgis, I guess it is still not noticed by team. 👀

**Reply by queenofcorgis** [reply]
@Neodevils Depending on the workload of the team, certain items may not be reviewed until a later date. Appreciate your patience!
